raiden/connection_manager.py

```diff
@@ -289,3 +320,23 @@ def open_channels(self):
             self.api.get_channel_list(token_address=self.token_address)
             if channel.isopen
         ]
+
+    @property
+    def min_settle_blocks(self):
+        """Returns the minimum necessary waiting time to settle all channels.
+        """
+        channels = self.api.get_channel_list(
+            token_address=self.token_address
+        )
+        timeouts = [0]
+        current_block = self.raiden.get_block_number()
+        for channel in channels:
+            if channel.state == CHANNEL_STATE_CLOSED:
+                since_closed = current_block - channel.external_state._closed_block
+            elif channel.state != CHANNEL_STATE_SETTLED:
```

Comment:
here you can drop the `elif` and merge with the `else` branch since they both have the same code.